runtime/compiler/optimizer/J9Optimizer.cpp

```diff
@@ -1027,14 +1027,7 @@ J9::Optimizer::optimizationStrategy(TR::Compilation *c)
       }
 
    TR_Hotness strategy = c->getMethodHotness();
-   if (strategy == warm && !c->getOption(TR_DisableCheapWarmOpts))
-      {
-      return cheapWarmStrategyOpts;
-      }
-   else
-      {
-      return j9CompilationStrategies[strategy];
-      }
+   return j9CompilationStrategies[strategy];
```

Comment:
Is there a cleanup task (separate PR) to free up the option bit for `TR_DisableCheapWarmOpts` ?